### PR TITLE
Register the Post Model to the admin site

### PR DESCRIPTION
**Pull request by Karthikeyantestpress**

### CODE REVIEW

File: mysite/blog/tests/test_admin.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+class PostAdminTest(TestCase):
+    def test_superuser(self):
+
+        password = "123"
+        test_superuser = User.objects.create_superuser(
+            "karthikeyanvenkat", "karthiktest51@gmail.com", password
+        )
+        c = Client()
+        c.login(username=test_superuser.username, password=password)
+        self.assertEqual(test_superuser.username, "karthikeyanvenkat")
```
Comment (waranlogesh) [inline mysite/blog/tests/test_admin.py:15]: Same. Tests not needed

File: mysite/blog/tests/test_admin.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+
+class PostAdminTest(TestCase):
+    def test_superuser(self):
+
+        password = "123"
+        test_superuser = User.objects.create_superuser(
+            "karthikeyanvenkat", "karthiktest51@gmail.com", password
+        )
+        c = Client()
```
Comment (waranlogesh) [inline mysite/blog/tests/test_admin.py:13]: image

File: mysite/blog/tests/test_admin.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+from django.contrib.auth.models import User
+
+
+class PostAdminTest(TestCase):
+    def test_superuser(self):
+
+        password = "123"
+        test_superuser = User.objects.create_superuser(
+            "karthikeyanvenkat", "karthiktest51@gmail.com", password
+        )
```
Comment (waranlogesh) [inline mysite/blog/tests/test_admin.py:12]: Use named arguments to be more explicit
```suggestion
        test_superuser = User.objects.create_superuser(
            username="karthikeyanvenkat", email="karthiktest51@gmail.com", password="123"
        )

```

**Review by waranlogesh — CHANGES_REQUESTED** [review]

File: mysite/blog/tests/test_admin.py (hunk 3 above)
Comment (Karthikeyantestpress) [reply mysite/blog/tests/test_admin.py]: Updated as the suggestion and Merged the PR

**Pull request merged**